eth/tracers/api_bor.go

```diff
@@ -115,7 +124,11 @@ func (api *API) traceBorBlock(ctx context.Context, block *types.Block, config *T
 	}
 
 	for indx, tx := range txs {
-		res.Transactions = append(res.Transactions, traceTxn(indx, tx))
+		if stateSyncPresent && indx == len(txs)-1 {
```

Comment:
Similar to the comment above, by default, we don't want to include state sync tx even when it presents in the block.

Reply:
Done